workers/ohsome_quality_analyst/ohsome/client.py

```diff
@@ -69,3 +62,36 @@ async def get_latest_ohsome_timestamp():
     timestamp_str = str(resp.json()["extractRegion"]["temporalExtent"]["toTimestamp"])
     timestamp = datetime.datetime.strptime(timestamp_str, "%Y-%m-%dT%H:%MZ")
     return timestamp
+
+
+def build_url(
+    layer,
+    endpoint: Optional[str] = None,
+    ratio: bool = False,
+) -> str:
+    """Build endpoint URL of ohsome API."""
+    ohsome_api = OHSOME_API.removesuffix("/")
```

Comment:
See comment [below](https://github.com/GIScience/ohsome-quality-analyst/pull/29/files?file-filters%5B%5D=.md&file-filters%5B%5D=.py#r636146069).

```suggestion
    ohsome_api = OHSOME_API.rstrip("/")
```